requirements.txt

```
@@ -37,6 +37,9 @@ transformers>=4.30.0,<=4.38.2
 trl>=0.7.11,<0.8.0
 wandb>=0.16.4,<0.17.0
 langchain-text-splitters
+# pinning tenacity to <8.4.0 due to langchain-text-splitters pulling it
+# https://github.com/instructlab/instructlab/issues/1389
+tenacity>=8.3.0,<8.4.0
```

tenacity just released 8.4.1 to fix this. So I don't think we need to do this anymore.

https://github.com/jd/tenacity/releases/tag/8.4.1

But if you want to exclude 8.4.0, then something like the following:

```suggestion
tenacity>=8.3.0,!=8.4.0
```

---

You also need to update your commit message (and PR title) to reflect the change.